src/pedalintandem.py

```diff
@@ -0,0 +1,114 @@
+import json
+from datetime import datetime
+from curl_cffi import requests
+from common.tz import IST
+from bs4 import BeautifulSoup
+
+BASE_URL = "https://www.pedalintandem.com"
+
+def _date(date_str):
+    return datetime.fromisoformat(date_str).astimezone(IST).isoformat()
+
+
+def fetch_events_links(session):
+	res = session.get(f"{BASE_URL}/experiences", impersonate="chrome")
+	soup = BeautifulSoup(res.text, 'html.parser')
+	event_divs = soup.find_all('div', class_ = 'single-experience')
+
+	# Fetch events from other pages.
+	load_more = soup.find('div', class_ = 'products-loadmore')
+	while load_more != None:
+		url = load_more.find('a').get('href')
```

Comment:
Switch to a single CSS selector here using `select` with a selector like `div.products-loadmore a[href^="/experiences/"]` to select all anchor tags that start with `/experiences/`. This gives category pages as well, so we can drop them after that.

You can also write a regex for the attribute selector, which bs4 supports..

Reply:
I don't understand this completely. Can you please rephrase it?